tracing-flame/src/lib.rs

```diff
@@ -235,6 +255,36 @@ where
             out: self.out.clone(),
         }
     }
+
+    /// Configures whether or not periods of time where no spans are entered
+    /// should be included
+    ///
+    /// Defaults to true
+    ///
+    /// Setting this feature to false can help with situations where you have
+    /// large periods of time spent idling or otherwise doing uninteresting work
+    /// where no spans are active. These empty samples can pollute the final
+    /// flamegraph and squish the work you care about into a much smaller
+    /// percentage of the overall graph, making it harder to navigate and in some
+    /// cases even hiding spans.
```

Comment:
Maybe
```suggestion
    /// When a large number of empty samples are recorded, the flamegraph
    /// may be harder to interpret and navigate, since the recorded spans will
    /// take up a correspondingly smaller percentage of the graph. In some
    /// cases, a large number of empty samples may even hide spans which
    /// would otherwise appear in the flamegraph.
```